tests/query/snql/test_query.py

```diff
@@ -11,159 +15,368 @@
 
 test_cases = [
     pytest.param(
-        "MATCH(blah)WHEREa<3COLLECT4-5,3*g(c),c",
+        "MATCH(e:Events) SELECT 4-5, c",
         Query(
             {},
-            None,
+            QueryEntity(
+                EntityKey.EVENTS, get_entity(EntityKey.EVENTS).get_data_model()
+            ),
+            selected_columns=[
+                SelectedExpression(
+                    "4-5",
+                    FunctionCall(None, "minus", (Literal(None, 4), Literal(None, 5))),
+                ),
+                SelectedExpression("c", Column("c", None, "c")),
+            ],
+        ),
+        id="Basic entity match",
+    ),
+    pytest.param(
+        "MATCH (e: Events) -[contains]-> (t: Transactions) SELECT 4-5, c",
+        Query(
+            {},
+            QueryEntity(
+                EntityKey.EVENTS, get_entity(EntityKey.EVENTS).get_data_model()
+            ),
+            selected_columns=[
+                SelectedExpression(
+                    "4-5",
+                    FunctionCall(None, "minus", (Literal(None, 4), Literal(None, 5))),
+                ),
+                SelectedExpression("c", Column("c", None, "c")),
+            ],
+        ),
+        id="Basic join match",
+    ),
+    pytest.param(
+        "MATCH (e: Errors) -[contains]-> (t: Transactions), (e: Errors) -[assigned]-> (ga: GroupAssignee) SELECT 4-5, c",
```

Comment:
Do we want to accept comma or just spaces between multiple relationships ? I am ok with either but we should converge on one of the two and correct the spec accordingly.

Reply:
I think commas between different relationships for readability. I'll ensure that's the case.